docs/marionette.application.md

```diff
@@ -2,21 +2,34 @@
 
 # Marionette.Application
 
-The `Backbone.Marionette.Application` object is the hub of your composite
-application. It organizes, initializes and coordinates the various pieces of your
-app. It also provides a starting point for you to call into from your HTML
-script block, or directly from your JavaScript files if you prefer to go that
-route.
+The **Application** is a container for the rest of your code. It is recommended
```

Comment:
I like the sentiment, but I wonder if I can communicate it in a different way. 'Bootstrapper' seems too colloquial. Nawm sayin?